unicore/hub/service/sso/templates/base.jinja2

```diff
@@ -6,10 +6,35 @@
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
         <title>{% block title %}{% endblock %}</title>
         <meta name="viewport" content="width=device-width; initial-scale=1.0; maximum-scale=1.0; user-scalable=0;" />
+        <link href="{{ request.static_url('unicore.hub:service/sso/static/css/style.css') }}" rel="stylesheet" type="text/css" />
     </head>
 
     <body>
+
         {% block body %}
+
+            <div class="header">
+            {% block header %}
+                {% if request.authenticated_userid %}
+                <a class='align-right' href="{{ request.route_path('user-logout', _query=request.query_string) }}">{{ _('Sign Out') }}</a>
```

Comment:
`pyramid_jinja2` ships a [`route_url`](http://docs.pylonsproject.org/projects/pyramid-jinja2/en/latest/#id4) filter for this kind of stuff, may be easier?